src/main/java/uk/gov/hmcts/reform/sscs/controller/SyaController.java

```diff
@@ -44,4 +44,23 @@
 
         return status(201).build();
     }
+
+
+    @ApiOperation(value = "submitDraftAppeal",
+            notes = "Creates a draft case from the SYA details",
+            response = String.class, responseContainer = "Appeal details")
+    @ApiResponses(value = {@ApiResponse(code = 201, message = "Submitted draft appeal successfully",
+            response = String.class)})
+    @RequestMapping(value = "/draft", method = POST, consumes = APPLICATION_JSON_VALUE)
```

Comment:
should be plural /drafts accordingly API definition best practices

Reply:
done